cli/cli.ts

```diff
@@ -1033,14 +1033,14 @@ function addCmd(name: string) {
     return name + (/^win/.test(process.platform) ? ".cmd" : "")
 }
 
-function buildPxtAsync(): Promise<string[]> {
+function buildPxtAsync(includeSourceMaps: boolean): Promise<string[]> {
```

Comment:
Sure. It does default to false right now, but I'll make it explicit in the declaration.